src/test/tasks.yaml

```diff
@@ -20,9 +23,13 @@ tasks:
 
   - name: create-deploy
     description: Test app used for UDS Core validation
+    inputs:
+      architecture:
+        description: "System architecture that the test-apps package should be built for."
+
```

Comment:
Default here should be `${UDS_ARCH}` I think to make sure that our github runners use an autodetected ARCH.